### DISCUSSION TITLE
[firebase_dynamic_links] ifl parameter does not take the user to specified url in iOS

### DISCUSSION THREAD

**Discussion by RamithRD** [firebase/flutterfire#6650] [Q&A]
Hi All,

I'm working on a Flutter app which currently only supports Android.

I've setup Dynamic Links and it's working as intended on Android and other platforms like desktop which I've made it work using the `ofl` parameter to take users to the same content on our website. I've also added the `ifl` parameter with the same url specified in the `ofl` but when I try open the dynamic link on Safari (iOS simulator), it says invalid dynamic link.

When generating the dynamic link, I did not pass the `IosParameters` since I'm not supporting iOS apps currently and I haven't registered an iOS app in Firebase console as well.

How do I setup my dynamic link to open my website when accessed from an iOS device?

This is the debug view of my dynamic link generated by concatenating both `ofl` and `ifl` parameters to the long url:

https://imgur.com/a/Mx5mbCD

Thank you!

**Comment by Vaytrex** [comment]
For anyone landing here after the Aug 2025 shutdown: there's no drop-in. You need a URL that branches on User-Agent (App Store / intent:// with browser_fallback_url / website) and skips redirecting crawlers so previews survive. Full write-up with the intent format: https://linkbah.app/firebase-dynamic-links-alternative
